### ISSUE TITLE
http:// and missing images

### ISSUE THREAD

**Issue by pgporada** [sensu-plugins/sensu-plugins-slack#39]
`http://` links should outright be `https://` to not have to rely on the webserver to do the redirect.

The images linked in the readme and handler are missing too.
```
README.md:[![Gem Version](https://badge.fury.io/rb/sensu-plugins-slack.svg)](http://badge.fury.io/rb/sensu-plugins-slack)
README.md:    "icon_url": "http://sensuapp.org/img/sensu_logo_large-c92d73db.png",
README.md:[Installation and Setup](http://sensu-plugins.io/docs/installation_instructions.html)
bin/handler-slack-multichannel.rb:      icon_url: 'http://sensuapp.org/img/sensu_logo_large-c92d73db.png',
bin/handler-slack.rb:      icon_url: slack_icon_url ? slack_icon_url : 'http://sensuapp.org/img/sensu_logo_large-c92d73db.png',
```

I was unable to find them on the sensu site after digging around a few file locations.

**Comment by eheydrick** [comment]
sensu logos can be found at https://github.com/sensu/sensu-logo. The broken links should get fixed. Another option is to upload the icon to the slack team and use `icon_emoji`.

**Comment by pgporada** [comment]
Will github care if I make a PR changing the image urls to pull from that repository? If so, could they hosted on https://sensuapp.org or put behind Fastly like the yum repository?

**Comment by eheydrick** [comment]
You can get the github raw URL that's served by fastly if you append _?raw=true_ to a repo URL. For example https://github.com/sensu/sensu-logo/blob/master/sensu1_flat%20white%20bg_png.png?raw=true is https://raw.githubusercontent.com/sensu/sensu-logo/master/sensu1_flat%20white%20bg_png.png. We can use that for the images.

**Comment by pgporada** [comment]
Thanks! I created a PR.

**Comment by majormoses** [comment]
addressed vi #40
